digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/RoadLinkService.scala

```diff
@@ -349,7 +349,7 @@ class RoadLinkService(val vvhClient: VVHClient, val eventbus: DigiroadEventBus,
     getRoadLinksAndChangesFromVVH(bounds, municipalities)._1
 
   def getViiteRoadLinksFromVVH(bounds: BoundingRectangle, roadNumbers: Seq[(Int, Int)], municipalities: Set[Int] = Set(),
-                               everything: Boolean, publicRoads: Boolean) : Seq[RoadLink] =
+                               everything: Boolean, publicRoads: Boolean = false) : Seq[RoadLink] =

```

Comment:
We're probably better of with code readability if these service level options don't have default values or at least these two either both have a default value or neither does.